src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/optimizer/randomized/IterativeImprovementBasedQueryOptimizer.java

```diff
@@ -7,7 +7,8 @@
 import se.liu.ida.hefquin.engine.queryproc.impl.optimizer.QueryOptimizationContext;
 import se.liu.ida.hefquin.engine.queryproc.impl.optimizer.rewriting.RuleInstances;
 import se.liu.ida.hefquin.engine.queryproc.impl.optimizer.utils.CostEstimationUtils;
-import se.liu.ida.hefquin.engine.utils.Pair;
+import se.liu.ida.hefquin.engine.queryproc.impl.optimizer.utils.PhysicalPlanWithCost;
+import se.liu.ida.hefquin.engine.queryproc.impl.optimizer.utils.PhysicalPlanWithCostUtils;
 
 public class IterativeImprovementBasedQueryOptimizer extends RandomizedQueryOptimizerBase

```

Comment:
The II branch was a branch out of this branch. I wanted to make a separate branch for the IterativeImprovement changes since you wanted the PR's to be more limited in scope, but I wanted to use something from a change I had made to the base class.

Reply:
Okay.